run_unittests.py

```diff
@@ -1782,6 +1830,21 @@ def test_dependency_invalid_method(self):
         '''
         self.assertMesonRaises(code, "Method.*configtool.*is invalid.*internal")
 
+    def test_objc_cpp_detection(self):
+        '''
+        Test that when we can't detect objc or objcpp, we fail gracefully.
+        '''
+        env = Environment('', self.builddir, self.meson_command,
+                          get_fake_options(self.prefix), [])
+        try:
+            objc = env.detect_objc_compiler(False)
```

Comment:
__[Flake8]__

[F841] local variable 'objc' is assigned to but never used


<!-- Comment by [Sider](https://sider.review) -->